usr_sbin/grml-config-root

```diff
@@ -40,7 +40,7 @@ case $retval in
           exec grml-setlang
         fi
         if [ "$MENU" = "grml-setkeyboard" ]; then
-          exec grml-setkeyboard
+          exec dpkg-reconfigure keyboard-configuration
```

Comment:
Maybe I'm holding it wrong, but tried to test this inside a VM by choosing `German (Austria)` keyboard, but doesn't seem to end up with such a keyboard setting then (tested inside tty1`), not claiming that my environment might be the problem, but I'd like further verification/testing yet before shipping that :)

Reply:
Thanks for testing :-/

Apparently console-setup doesn't bother to apply the new settings after debconf exits. I'll spare you the details 😱